### ISSUE TITLE
documentation / example use case

### ISSUE THREAD

**Issue by Sophrinix** [Ziggeo/ZiggeoRubySdk#2]
I'm glad to see this gem, but this is a case of RTFC. 

Could someone post a quick example of how to use this in a rails view? I'm currently reading through the code, so I might become obvious, but the documentation does not exist. 

Again, thanks for writing this and making it open source.

**Comment by oliverfriedmann** [comment]
Fair enough. Most of Ziggeo is client-side (e.g. just HTML+CSS+JS), so the documentation on that is not Ruby-specific. Please see http://ziggeo.com/docs and http://ziggeo.com/reference

Both links also contain documentation on how to add the Ruby server SDK to your application as well as which methods exists (essentially CRUD operations on video objects) and how to call them in Ruby.

Totally agree with you that we should add docs to this gem!

**Comment by AlexVKO** [comment]
Please, examples use in rails views and etc...

**Comment by oliverfriedmann** [comment]
Added some documentation, please let me know whether it is helpful.

**Comment by sambua** [comment]
Very useful post how you can use [Rails and Ziggeo](https://www.sitepoint.com/video-uploads-with-rails-and-ziggeo/)
